saleor/graphql/account/mutations/jwt.py

```diff
@@ -69,11 +71,20 @@ class Meta:
     )
     user = graphene.Field(User, description="A user instance.")
 
+    @classmethod
+    def _retrieve_user_from_credentials(cls, email, password) -> Optional[models.User]:
+        user = models.User.objects.filter(email=email).first()
+        if not user:
+            return None
+        if not user.is_active:
+            return None
+        if not user.check_password(password):
+            return None
+        return user
```

Comment:
A bit shorter and should do the job:

```suggestion
        user = models.User.objects.filter(email=email, is_active=True).first()
        if user and user.check_password(password):
            return user
        return None
```

What do you think?